host/xtest/regression_4000.c

```diff
@@ -24,6 +24,9 @@
 
 #include <assert.h>
 
+#if defined(CFG_CRYPTOLIB_NAME_mbedtls)
```

Comment:
i think you can remove the conditioning.

Reply:
ok